0-module-and-module-source.md

```diff
@@ -48,33 +48,39 @@ of an agent cluster.
 ```ts
 type ImportSpecifier = string;
 
-type ImportHook = (specifier: ImportSpecifier, importMeta: object) =>
+type ImportHook = (this: ModuleHandler, specifier: ImportSpecifier) =>
   Promise<Module>;
 
+type ImportMetaHook = (this: ModuleHandler, importMeta: any) => any;
```

Comment:
`importMeta` is always an object with `__proto__` set to undefined by the spec. Not sure who to represent it here... but the fact that this object can be augmented is a given for this API.

Reply:
I’ll add a `type ImportMeta = { __proto__: null } & any;`